.travis.yml

```diff
@@ -33,7 +33,7 @@ before_install:
   - npm config set prefix $HOME/.npm-prefix --global
 
 install:
-  - time mvn -q -T 2C -DskipTests clean install
+  - time mvn -q -T 2C -DskipTests -PHDP-2.5.0.0 clean install
```

Comment:
I believe the rationale for having travis run without the 2.5.0.0 profile and full-dev build with it is to ensure that we build with both per PR (since both are required for a +1).  I would be more comfortable with a follow-on task to make a proper travis matrix where both are run in travis per build.

Reply:
Just so I'm completely clear, by "build matrix" I mean [this](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix)

Reply:
That works.